### PR TITLE
37rc3

### PR DESCRIPTION
**Pull request by zero-24**
cc @rdeutz

### CODE REVIEW

File: www/core/test/list_test.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <extensionset name="Joomla Core Test Updateserver" description="The Joomla Core Updateserver for Tests of Alpha, Beta und RC Releases">
```
Comment (brianteeman) [inline www/core/test/list_test.xml]: and not und ;)

Comment (zero-24) [inline www/core/test/list_test.xml]: Thanks original this was all german so i have missed this on taking over here. Thanks for spotting

Comment (brianteeman) [inline www/core/test/list_test.xml]: update server should be two words as well :)

Comment (zero-24) [inline www/core/test/list_test.xml]: done btw you can also do that using the edit tool from github as you have write access to that repo.

Comment (brianteeman) [inline www/core/test/list_test.xml]: But then your English will never improve

Comment (zero-24) [inline www/core/test/list_test.xml]: I think this was not about good vs. bad english in this case but in general you are correct 😄

**Comment by zero-24** [comment]
@rdeutz please change the version file to https://github.com/joomla/joomla-cms/blob/staging/libraries/cms/version/version.php#L41

`const DEV_LEVEL = '0-dev';` this should results into `Joomla_3.7.0-dev-dev-Update_Package.zip`

**Comment by zero-24** [comment]
Thanks.

**Pull request merged**